### PR TITLE
3.0.3 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.3 (2019-04-25)
+
+- Fixed flyout component to not close when infinite scroll "Load more" button is clicked. [#30](https://github.com/blackbaud/skyux-flyout/pull/30)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: This describes the issue we addressed, but not the actual change we made to the flyout. Including the infinite scroll's behavior in the flyout's changelog feels wrong?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: How's that?

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: What about "Fixed flyout component to remain open when users click infinite scroll "Load more" button." ?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: @Blackbaud-DeniseCisneros sorry, refresh your screen - I just removed "infinite scroll" all together and made it more generic.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.0.3 (2019-04-25)
+
+- Fixed flyout component to not close when clicking on internal components. [#30](https://github.com/blackbaud/skyux-flyout/pull/30)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: My bad. Went to the tab I had open.

Fixed flyout component to remain open when users click internal components.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: That looks good to me.

**Pull request merged**